### PR TITLE
Cloud Enterprise FAQs

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://mattermost.atlassian.net/browse/MM-33539

Updated:
- Overview > Mattermost Cloud Overview > Frequently Asked Questions (FAQs)
   - Added 3 FAQs including:
      - Is Mattermost Cloud Enterprise a dedicated instance run on AWS systems?
      - Is customer data in Mattermost-Cloud Enterprise fully encrypted?
      - Are S3-managed keys used for server-side encryption?

### CODE REVIEW

File: source/overview/mattermost-cloud-overview.rst [hunk 1]
```diff
@@ -66,3 +66,20 @@ What are the alternatives, and why should customers choose Mattermost Cloud?
 Other SaaS offerings are closed-source and focus on a very broad usecase. Mattermost Cloud is built on our open source core offering, allowing customers to view our source code and have a high degree of transparency into Mattermost development processes. Mattermost provides a DevOps-centric offering and has deep integrations and workflows that help builders and operators be more effective.
 
 Mattermost Cloud Enterprise is deployed in a private environment within an AWS VPC dedicated to a single customer. Other SaaS offerings are multi-tenant, meaning customers share resources.
+
+Is Mattermost Cloud Enterprise a dedicated instance run on AWS systems?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Yes, Mattermost Cloud Enterprise is a dedicated Mattermost environment running in a dedicated AWS account with separate infrastructure for that customer specifically, i.e. separate database, separate VMs, separate Kubernetes cluster.
+
+Is customer data in Mattermost-Cloud Enterprise fully encrypted?
```
Comment (jasonblais) [inline source/overview/mattermost-cloud-overview.rst:75]:
```suggestion
Is customer data in Mattermost Cloud Enterprise fully encrypted?
```

File: source/overview/mattermost-cloud-overview.rst [hunk 2]
```diff
@@ -66,3 +66,20 @@ What are the alternatives, and why should customers choose Mattermost Cloud?
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Mattermost utilizes AWS-provided functionality to enable encryption-at-rest for both databases and file stores. See `Encrypting Amazon RDS resources - Amazon Relational Database Service <https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html>`__ and `Protecting data using server-side encryption - Amazon Simple Storage Service <https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html>`__ for details.
```
Comment (jasonblais) [inline source/overview/mattermost-cloud-overview.rst:78]: It's not immediately obvious to me if the answer is yes or 'partially' yes?

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: source/overview/mattermost-cloud-overview.rst (hunk 2 above)
Comment (cwarnermm) [reply source/overview/mattermost-cloud-overview.rst:78]: @jasonblais - It's not immediately obvious to me, either. I believe that the encryption-at-rest satisfies the condition of "fully encrypted". If you agree, then I'll update this FAQ answer to start with "Yes."

Comment (wiersgallak) [reply source/overview/mattermost-cloud-overview.rst:78]: @cwarnermm How about we change the question?  "How is customer data in Mattermost Cloud Enterprise encrypted?"

Comment (cwarnermm) [reply source/overview/mattermost-cloud-overview.rst:78]: I like that, @wiersgallak! Thanks!

File: source/overview/mattermost-cloud-overview.rst [hunk 3]
```diff
@@ -66,3 +66,20 @@ What are the alternatives, and why should customers choose Mattermost Cloud?
 Other SaaS offerings are closed-source and focus on a very broad usecase. Mattermost Cloud is built on our open source core offering, allowing customers to view our source code and have a high degree of transparency into Mattermost development processes. Mattermost provides a DevOps-centric offering and has deep integrations and workflows that help builders and operators be more effective.
 
 Mattermost Cloud Enterprise is deployed in a private environment within an AWS VPC dedicated to a single customer. Other SaaS offerings are multi-tenant, meaning customers share resources.
+
+Is Mattermost Cloud Enterprise a dedicated instance run on AWS systems?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Yes, Mattermost Cloud Enterprise is a dedicated Mattermost environment running in a dedicated AWS account with separate infrastructure for that customer specifically, i.e. separate database, separate VMs, separate Kubernetes cluster.
+
+Is customer data in Mattermost Cloud Enterprise fully encrypted?
```
Comment (cwarnermm) [inline source/overview/mattermost-cloud-overview.rst:75]:
```suggestion
Is customer data in Mattermost Cloud Enterprise fully encrypted?
```
```suggestion
Is customer data in Mattermost Cloud Enterprise fully encrypted?
```
```suggestion
How is customer data in Mattermost Cloud Enterprise encrypted?
```

File: source/overview/mattermost-cloud-overview.rst (hunk 2 above)
Comment (jasonblais) [reply source/overview/mattermost-cloud-overview.rst:78]:
```suggestion
Mattermost uses AWS-provided functionality to enable encryption-at-rest for both databases and file stores. See `Encrypting Amazon RDS resources - Amazon Relational Database Service <https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html>`__ and `Protecting data using server-side encryption - Amazon Simple Storage Service <https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html>`__ for details. 
```

**Review by jasonblais — APPROVED** [review]
One minor suggestion, else LGTM!

**Review by wiersgallak — APPROVED** [review]
FYI, this section will be changing again with the packaging changes.  I am ok to approve these for now, but they will be changing in the next few weeks.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**